src/app/app.component.html

```diff
@@ -15,6 +15,7 @@
                 [choose]="chosenItem"
                 [docs]="inputVal.docs"
                 [examples]="inputVal.examples"></jsdaddy-open-source-options>
+            <jsdaddy-open-source-footer></jsdaddy-open-source-footer>
```

Comment:
Change to `<jsdaddy-open-source-footer></jsdaddy-open-source-footer> -> <jsdaddy-open-source-footer />`